Expense Tracker/script.py

```diff
@@ -0,0 +1,97 @@
+from tkinter import ttk
+from tkinter import messagebox
+from tkinter import *
+from tkinter.ttk import Notebook
+from tkcalendar import DateEntry
+import sqlite3
+
+
+def Addexpense():
+    x = Edate.get()
+    y = Title.get()
+    z = Eexpense.get()
+    data = [x,y,z]
+    TVExpense.insert('','end',values=data)
+    with db:
+        c = db.cursor()
+        c.execute("INSERT INTO expense(Dates, Items, Expense) VALUES(?,?,?)",(x,y,z))
+       
+
+
+def show():
+    connt=sqlite3.connect('expense.db')
+    cursor=connt.cursor()
+    cursor.execute("SELECT * FROM expense")
+    for row in cursor.fetchall():
+        print(row)
```

Comment:
instead of printing, show the results in the table in the GUI

Reply:
In the GUI the items that we add is visible. Then how can I show the entire list??